etc/ci/upload_nightly.sh

```diff
@@ -12,7 +12,12 @@ usage() {
 
 
 upload() {
-    s3cmd put "${2}" "s3://servo-builds/nightly/${1}/"
+    local s3path filename
+    s3path="s3://servo-builds/nightly"
```

Comment:
Let's call this variable `nightly_upload_dir`. Let's also declare it as readonly:
```
local -r nightly_upload_dir="..."
```